api/src/main/resources/org/openmrs/liquibase/snapshots/core-data/liquibase-core-data-2.6.x.xml

```diff
@@ -5218,4 +5218,23 @@
       <column name="role" value="System Developer"/> 
     </insert> 
   </changeSet>
+	<changeSet author="isaiah (generated)" id="202405201900-1">
+		<insert tableName="scheduler_task_config">
+			<column name="task_config_id" valueNumeric="3"/>
```

Comment:
Instead of automatically creating this task, i would prefer some sort of documentation for how to create the task via the user interface such that its gets configured by whoever needs this functionality.

Reply:
Could you suggest where I can add the documentation.

Reply:
You can start by including it in this list: https://openmrs.atlassian.net/wiki/spaces/docs/pages/25520634/Administering+Scheduled+Tasks\
And then provide a link to it that directs one to a page with its details.

Reply:
Done